### PR TITLE
Added the endpoints for getting client position mode

### PR DESCRIPTION
**Pull request by dinmeiri**

### CODE REVIEW

File: python/ccxt/bybit.py [hunk 1]
```diff
@@ -541,6 +541,27 @@ def change_position_mode(self, is_hedge_mode, symbol=None, pair=None):
...
+            positions = self.safe_value(response, 'result')
+            for _position in positions:
+                position = self.safe_value(_position, "data", _position)
+                if position:
+                    mode = self.safe_value(position, "mode")
+                    if self.safe_value(position, "size", 0):
+                        open_positions_mode_count[mode] += 1
+                    all_symbols_mode_count[mode] += 1
+            if open_positions_mode_count["BothSide"] == open_positions_mode_count["MergedSingle"]:
+                return all_symbols_mode_count["BothSide"] > all_symbols_mode_count["MergedSingle"]
```
Comment (dinmeiri) [inline python/ccxt/bybit.py:560]: We still need to decide what to do when user has no open positions

File: python/ccxt/bybit.py [hunk 2]
```diff
@@ -541,6 +541,27 @@ def change_position_mode(self, is_hedge_mode, symbol=None, pair=None):
...
+            self.load_markets()
+            response = self.privateGetPrivateLinearPositionList()
+            open_positions_mode_count = {"BothSide": 0, "MergedSingle": 0}
+            all_symbols_mode_count = {"BothSide": 0, "MergedSingle": 0}
+            positions = self.safe_value(response, 'result')
+            for _position in positions:
+                position = self.safe_value(_position, "data", _position)
+                if position:
+                    mode = self.safe_value(position, "mode")
+                    if self.safe_value(position, "size", 0):
```
Comment (giladcornix) [inline python/ccxt/bybit.py:556]: sounds like a safe int

File: python/ccxt/bybit.py [hunk 3]
```diff
@@ -541,6 +541,27 @@ def change_position_mode(self, is_hedge_mode, symbol=None, pair=None):
         except NotChanged:
             pass
 
+    def get_position_mode(self):
+        _type = self.safe_string(self.options, 'defaultType')
+        if _type == "linear":
+            self.load_markets()
+            response = self.privateGetPrivateLinearPositionList()
+            open_positions_mode_count = {"BothSide": 0, "MergedSingle": 0}
```
Comment (giladcornix) [inline python/ccxt/bybit.py:549]: wrap the counting phase into func

File: python/ccxt/bybit.py [hunk 4]
```diff
@@ -541,6 +541,27 @@ def change_position_mode(self, is_hedge_mode, symbol=None, pair=None):
...
+            all_symbols_mode_count = {"BothSide": 0, "MergedSingle": 0}
+            positions = self.safe_value(response, 'result')
+            for _position in positions:
+                position = self.safe_value(_position, "data", _position)
+                if position:
+                    mode = self.safe_value(position, "mode")
```
Comment (giladcornix) [inline python/ccxt/bybit.py:555]: safe_string

File: python/ccxt/bybit.py (hunk 1 above)
Comment (giladcornix) [reply python/ccxt/bybit.py:560]: Let's make it simpler and take hedge mode if there's more than 3 symbols with bothside in that case.

**Review by giladcornix — CHANGES_REQUESTED** [review]

**Review by giladcornix — APPROVED** [review]

**Pull request merged**